test-common/src/main/kotlin/SøknadsbehandlingTestData.kt

```diff
@@ -557,3 +565,240 @@ fun søknadsbehandlingLukket(
+ */
+fun nySøknadsbehandling(
+    clock: Clock = fixedClock,
+    stønadsperiode: Stønadsperiode = stønadsperiode2021,
+    sakOgSøknad: Pair<Sak, Søknad.Journalført.MedOppgave>,
+): Pair<Sak, Søknadsbehandling.Vilkårsvurdert> {
+    return sakOgSøknad.let { (sak, søknad) ->
+        val søknadsbehandling = Søknadsbehandling.Vilkårsvurdert.Uavklart(
+            id = UUID.randomUUID(),
+            opprettet = Tidspunkt.now(clock),
+            sakId = sak.id,
+            saksnummer = sak.saksnummer,
+            søknad = søknad,
+            oppgaveId = søknad.oppgaveId,
+            behandlingsinformasjon = Behandlingsinformasjon.lagTomBehandlingsinformasjon(),
+            fnr = sak.fnr,
+            fritekstTilBrev = "",
+            stønadsperiode = stønadsperiode,
+            grunnlagsdata = Grunnlagsdata.IkkeVurdert,
+            vilkårsvurderinger = when (sak.type) {
+                Sakstype.ALDER -> {
+                    Vilkårsvurderinger.Søknadsbehandling.Alder.ikkeVurdert()
+                }
+                Sakstype.UFØRE -> {
+                    Vilkårsvurderinger.Søknadsbehandling.Uføre.ikkeVurdert()
+                }
+            },
+            attesteringer = Attesteringshistorikk.empty(),
+            avkorting = AvkortingVedSøknadsbehandling.Uhåndtert.IngenUtestående.kanIkke(),
+            sakstype = sak.type,
+        )
+
+        // legg til for å oppdater stønadsperiode
+        val førOppdatertStønadsperiode = sak.copy(
+            søknader = sak.søknader.filterNot { it.id == søknad.id } + søknad, // replace hvis søknaden allerede er lagt til (f.eks hvis man først oppretter bare sak + søknad)
+            søknadsbehandlinger = sak.søknadsbehandlinger + søknadsbehandling,
+        )
+
+        // oppdater stønadsperiode via sak for å sjekke gyldigheten
+        val etterOppdatertStønadsperiode = førOppdatertStønadsperiode.oppdaterStønadsperiodeForSøknadsbehandling(
+            søknadsbehandlingId = søknadsbehandling.id,
+            stønadsperiode = stønadsperiode,
+            clock = clock,
+            formuegrenserFactory = formuegrenserFactoryTestPåDato(LocalDate.now(clock)),
+        ).getOrFail()
+
+        sak.copy(
+            søknader = sak.søknader.filterNot { it.id == søknad.id } + søknad, // replace hvis søknaden allerede er lagt til (f.eks hvis man først oppretter bare sak + søknad)
+            søknadsbehandlinger = sak.søknadsbehandlinger.filterNot { it.id == etterOppdatertStønadsperiode.id } + etterOppdatertStønadsperiode,
+        ) to etterOppdatertStønadsperiode
+    }
+}
+
+fun vilkårsvurdertSøknadsbehandlingAlder(
+    clock: Clock = fixedClock,
+    stønadsperiode: Stønadsperiode = stønadsperiode2021,
+    sakOgSøknad: Pair<Sak, Søknad.Journalført.MedOppgave> = nySakAlder(
+        clock = clock,
+    ),
+    customGrunnlag: List<Grunnlag> = emptyList(),
+    customVilkår: List<Vilkår> = emptyList(),
```

Comment:
Burde vi asserte at vilkår og grunnlag ikke er duplikate? Fant noen tester med det :man_shrugging: som sannsynligvis ikke skulle vært slik

Reply:
Sikkert greit å håndheve, spesielt for vilkår.